dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1446,10 +1449,24 @@ a.download-video {
   #answerdest {
     width: 240px;
   }
+  .answerlist {
+    h4 {
+      position: relative;
+      top: 25%;
+      @include yTranslation(-50%);
+    }
+  }
   #answers {
     list-style-type: none;
     margin-left: 0px;
   }
+  .answerslot {
+    h4 {
```

Comment:
My mistake, this was a change I had made before I knew we were taking out the h4's from the multi puzzles. This change still works though, as it only affects one of the match puzzles that use h4. And it looks better, in my opinion. The match puzzle I'm referring to is featured in the picture I uploaded in the description.